examples/enforcement/breach.json

```diff
@@ -0,0 +1,397 @@
+                  51.38713965241118
+                ],
+                [
+                  0.5062454938888572,
+                  51.387138815488186
+                ],
+                [
+                  0.5063232779502891,
+                  51.38716141240323
+                ],
+                [
+                  0.5063809454441092,
+                  51.387173966240084
+                ],
+                [
+                  0.5064935982227345,
+                  51.38722836616
+                ],
+                [
+                  0.5064211785793326,
+                  51.387287787537105
+                ],
+                [
+                  0.5063098669052146,
+                  51.387310384378566
+                ],
+                [
+                  0.5061100423336052,
+                  51.387245941504915
+                ]
+              ]
+            ]
+          },
+          "properties": {
+            "label": "1",
+            "area.hectares": 0.030997000000000004,
+            "area.squareMetres": 309.97,
+            "planx_user_action": "Drew a custom boundary"
+          }
+        },
+        "area": {
+          "squareMetres": 309.97
+        }
+      }
+    }
+  },
+  "responses": [
+    {
+      "question": "Are you reporting any of the following?",
+      "responses": [
+        {
+          "value": "No, none of the above"
+        }
+      ],
+      "metadata": {}
+    },
+    {
+      "question": "Is the property in Medway?",
+      "responses": [
+        {
+          "value": "Yes"
+        }
+      ],
+      "metadata": {
+        "autoAnswered": true
+      }
+    },
+    {
+      "question": "Has the property or site already been granted planning permission for the works you're reporting?",
+      "responses": [
+        {
+          "value": "Yes, it's been granted planning permission but I still think it's a breach"
+        }
+      ],
+      "metadata": {}
+    },
+    {
+      "question": "Enter your contact details",
+      "responses": [
+        {
+          "value": "Ebenezer Scrooge 0123456789 scrooge@waltdisney.com"
+        }
+      ],
+      "metadata": {}
+    },
+    {
+      "question": "Enter your address",
+      "responses": [
+        {
+          "value": "16 Bayham Street, Camden Town, London, NW1 0JU, UK"
+        }
+      ],
+      "metadata": {}
+    },
+    {
+      "question": "Is the breach currently in progress?",
+      "responses": [
+        {
+          "value": "Yes"
+        }
+      ],
+      "metadata": {}
+    },
+    {
+      "question": "Select the breach you are reporting",
+      "responses": [
+        {
+          "value": "New or modified outbuilding (e.g. garage or shed)"
+        }
+      ],
+      "metadata": {}
+    },
+    {
+      "question": "Describe the work suspected to be a breach",
+      "responses": [
+        {
+          "value": "Erection of a library in the rear garden"
+        }
+      ],
+      "metadata": {}
+    },
```

Comment:
"Select the breach you are reporting" and "Describe the work suspected to be a breach" seem like good candidates for `data.proposal.projectType` and `data.proposal.description` or similar, but are not currently built to use schema variables in the editor.